### PR TITLE
Project finished waiting for review

### PR DESCRIPTION
**Pull request by Miguelus373**
We had a Webhint error that we don't know how to fix. It is the Forbidden code point U+007f.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,9 +4,645 @@
...
+    <header class="header flex">
+        <!-- Navbar -->
+        <nav class="navbar flex">
+            <div class="left-nav flex">
+                <i class="fas fa-bars"></i>
+                <i class="fas fa-search"></i>
+            </div>
+            <div class="middle-nav">
+                <img src="./img/logo.png" alt="logo">
+            </div>
```
Comment (expjazz) [inline index.html]: This logo needs to be a link 
image

File: style.css [hunk 2]
```diff
@@ -0,0 +1,857 @@
...
+  content: "Most Popular";
+  width: 130%;
+  height: 0.1rem;
+  position: absolute;
+  top: -4rem;
+  left: -10rem;
+  background-color: black;
+  font-size: 1.6rem;
+  font-weight: 700;
+}
```
Comment (expjazz) [inline style.css]: The width of this line should be the same from the items below.
| Yours | Original |
|:---:|:---:|
| image | image |

**Review by expjazz — CHANGES_REQUESTED** [review]
## Status: [Changes requested :recycle: ]

#### Hello Team. You did great in this project. :clap: 

The page looks really great, and more than that, the code is also good. You showcased that you know how to use grids, flexbox, floats, and CSS pseudo-elements. The code is clean and very easy to understand :100: 

There are only two details that need to be fixed (I specified below, please check the inline comments).

You also have linter errors. Kindly fix that as well  (check  them locally)

### Please make the required changes and resubmit for a new code review once you are ready
Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

File: style.css (hunk 2 above)
Comment (Miguelus373) [reply style.css]: Thank you for the review! We fixed those two details

**Review by kitpao — CHANGES_REQUESTED** [review]
# 👋 Hello @Miguelus373 and @solbencas ,

You did a nice job doing the requested changes 😎

About your linters issue, it is triggered because you are using Unicode for a forbidden character, U+007f means the "delete" key, I see that the file that is triggerig the error was most probably generated when you ran your linters locally, please take a look to this point to fix it:

- [ ] **[MANDATORY]** Files not related to the project should not be committed and added to .gitignore (e.g. your hint-report folder). This is also what is triggering the linter error, so you have to: remove it from your repo (put it in another place in your computer), push your project without the folder, add the folder to gitignore, push it, return your hint-report folder to your folder and make sure the folder doesn't appear in your github repo.
image

I hope this was helpful for you, for now
### ♻️ Status: Your project requires some additional changes ♻️
kindly do them and submit for another code review

___
##### 👉 If you have any doubts you can contact me in slack as @kitpao
##### 👉 You can find me in your completition form as Kitzia Paola Vidal Marroquin

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__

**Comment by ghost** [comment]
thank you for the instructions, we just fixed webhint error and now we are ready for our last code review.

**Review by expjazz — APPROVED** [review]
### Last review indeed. Status: Project Approved :clap: 

Go ahead and merge :+1: 

#### Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Pull request merged**